setup_build_env.sh

```diff
@@ -18,10 +19,11 @@ if [ "$(uname)" != "Darwin" ]; then
   export THRIFT_HOME=$BUILD_DIR/thirdparty/installed
 fi
 
+export GTEST_HOME=$BUILD_DIR/thirdparty/$GTEST_BASE_DIR
```

Comment:
The CMake setup doesn't provide a "make install" target, similarly Homebrew doesn't allow users to install it. I don't have 100% context why that is